yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
This is here because without it Yarn uses [PNP](https://yarnpkg.com/features/pnp) but we need to do a bit of manual copying of images out of `node_modules` so Rails can pick them up.

Reply:
I've seent his upgrade a few time and it also has the line `yarnPath: .yarn/releases/yarn-4.1.1.cjs` is this something we want to add to the template?

Reply:
That's set by the `run "yarn set version stable"` which runs just after this happens in the template - both lines _should_ be present after a successful run.

Reply:
I wondered if setting it upfront would skip the need to run `yarn set version stable` never tried it either way 👌